tests/unittests/test_handler/test_handler_mounts.py

```diff
@@ -183,6 +185,56 @@ def device_name_to_device(self, path):
 
         return dev
 
+    @mock.patch('cloudinit.util.kernel_version')
+    @mock.patch('cloudinit.util.get_mount_info')
+    def test_swap_creation_method(self, m_kernel_version, m_get_mount_info):
+        log = mock.MagicMock()
+        m_kernel_version.return_value = (3, 18)
+        m_get_mount_info.return_value = ["", "xfs"]
+        fstab = '/swap.img swap swap defaults 0 0\n'
+
+        with open(cc_mounts.FSTAB_PATH, 'w') as fd:
+            fd.write(fstab)
+        cc = {'swap': ['filename: /swap.img', 'size: 512', 'maxsize: 512']}
+        cc_mounts.handle(None, cc, self.mock_cloud, self.mock_log, [])
+
+        swap_log = 'Creating swapfile in /swap.img on fstype xfs using dd'
+        util.multi_log('{0}\n'.format(swap_log), log=log)
+        self.assertEqual((mock.ANY, swap_log), log.log.call_args[0])
```

Comment:
These lines look like they're just testing themselves, rather than any behaviour in the unit under test.